pkg/capabilities/consensus/ocr3/datafeeds/data_feeds_types.proto

```diff
@@ -9,6 +9,8 @@ message DataFeedsOutcomeMetadata {
 }
 
 message DataFeedsMercuryReportInfo {
-  uint32 timestamp = 1;
-  double price = 2;
+  // DEPRECATED = 1;
+  // DEPRECATED = 2;
```

Comment:
nit: proto supports the reserved keyword for this: https://protobuf.dev/programming-guides/proto3/#fieldreserved

Reply:
oh nice, let me change :)